src/nuclio/projects/project/functions/version/version-monitoring/version-monitoring.less

```diff
@@ -89,7 +89,6 @@
                 }
 
                 .ncl-monitoring-error-logs {
-                    height: 0px;
```

Comment:
What bug does this fix?

Reply:
It was a bug with displaying of Error block on Status screen.